app/controllers/api/v1/mapping_terms_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+###
+# @description: Place all the actions related to mappings
+###
+class Api::V1::MappingTermsController < ApplicationController
+  before_action :authorize_with_policy
+
+  def update
+    @mapping.update_mapped_terms(params[:mapping_term][:mapped_terms]) if params[:mapping_term][:mapped_terms].present?
+    @mapping.update!(permitted_params)
+
+    render json: @mapping, inlcude: %i[mapped_terms]
+  end
+
+  private
+
+  ###
+  # @description: Execute the authorization policy
+  ###
+  def authorize_with_policy
+    authorize mapping_term
+  end
+
+  ###
+  # @description: Get the current model record
+  # @return [ActiveRecord]
```

Comment:
Totally not a biggie, but the returned type is `MappingTerm`, `ActiveRecord` is a module.